addons-web-sdk/samples/animation-next-js/next.config.mjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  basePath: '/meet/animation-next-js',
+  distDir: '../dist/animation-next-js',
+  output: 'export',
```

Comment:
Done

Reply:
These actually don't work with `output: export`, and I decided that I didn't want them in the sample code anyway (since I don't want people to copy-paste this into their own projects if they're just copying our samples)

So I moved them into `/dist`. thanks!!